README.md

```diff
@@ -88,108 +75,172 @@ By default, entries in the OpenAPI Explorer pane are sorted alphabetically. If y
+- Validate that the implementation of the API conforms to its established contract: Scan checks all responses against the OpenAPI definition and detects unexpected responses and data leaks.
 
-![Details for specific issues](https://github.com/42Crunch/vscode-openapi/blob/master/images/Details%20for%20specific%20issues.gif?raw=true)
+![](https://img.shields.io/badge/Learning%20More!-Blue) Watch this [video](https://42crunch.com/free-user-faq/?utm_campaign=IDE&utm_medium=referral&utm_source=vscode#Free-API-Security-Testing-FAQs-4) to learn more about 42Crunch Scan.
 
-3. For a quick overall look, check the counts in the Status Bar. The different icons match the severity of the issue:
+APIs which thoroughly enforce compliance to an established contract are far more resilient to all types of attacks.
 
-- Error: critical or high
-- Warning: medium
-- Info: low
+![](https://img.shields.io/badge/Important-red)  You should only use 42Crunch Scan against APIs that you own, but not those of third parties.
 
-![List of API Sec Issues](https://github.com/42Crunch/vscode-openapi/blob/master/images/List%20of%20API%20Sec%20Issues.gif?raw=true)
+### Launching 42Crunch Scan 
 
-4. Click the icons in the Status Bar to open the PROBLEMS panel and scroll through the issue titles. The issues are ordered from most to least severe, so it is easy for fix the worst offenders first. Note that recommendations are not listed in the PROBLEMS panel.
-5. Click on an issue in the PROBLEMS panel to jump to view it inline in your API definition.
+Scans are run at the operation level only. We recommend you use the 42Crunch CLI to run scans. The alternative is to run a docker image locally. 42Crunch customers can also leverage our [scand manager](https://github.com/42Crunch/scand-manager), by deploying an API-driven scan engine on Kubernetes.
 
-### Fix issues
+In order to run a scan, you will need : 
 
-Many of the issues reported by the Security Audit have Fixes associated with them. These are code snippets that you can get inserted into the OpenAPI file and then customize with the appropriate value.
+- **A credential** : most likely, your API is using some form of authentication, like an API Key or token. You need a valid credential to provide to the scan engine. 
 
-#### Individual fixes
+- **The URL** when the API is deployed.  
 
-1. Perform the Security Audit by clicking the **42c** button.
-2. When you click some of the error lines in your OpenAPI file, you will see the QuickFix icon show up on the left of the line.
-3. Click the icon and select the QuickFix to apply.
-4. Tweak the values in the inserted code snippet as you see fit.
+  ![](https://img.shields.io/badge/Warning-orange)  We recommend that you do <u>not</u> target a production system. While the tool does not try to inject malicious payloads, it is possible that the API implementation is not resilient enough to handle the tests and may crash or behave unexpectedly.
```

Comment:
I think we've elsewhere been much more strict and said that they MUST not target production. I think this may even be in our Ts&Cs. Of course, this is now in the context of freemium where they have not yet had to sign up to our Ts&Cs, so perhaps our approach could be more lenient then? In  any case, something to think about in terms of the wording here.